fem/fespace.cpp

```diff
@@ -3678,4 +3678,11 @@ FiniteElementCollection *FiniteElementSpace::Load(Mesh *m, std::istream &input)
    return r_fec;
 }
 
+ElementDofOrdering GetEVectorOrdering(const FiniteElementSpace& fes)
+{
+   return UsesTensorBasis(fes)?
+          ElementDofOrdering::LEXICOGRAPHIC:
+          ElementDofOrdering::NATIVE;
+}
```

Comment:
@YohannDudouit responding to

I use this  at `bilininteg_elasticity_pa.cpp` and `bilinearform_ext.cpp`

Reply:
in reference to [this](https://github.com/mfem/mfem/pull/3806#discussion_r1380765000) comment

Reply:
So now it looks like
[https://github.com/mfem/mfem/blob/4a38577e3f80d9252c1d81462a614a2206c4c3a6/fem/bilinearform_ext.cpp#L255-L259](https://github.com/mfem/mfem/blob/4a38577e3f80d9252c1d81462a614a2206c4c3a6/fem/bilinearform_ext.cpp#L255-L259)